cmd/analyzer/parse_args.go

```diff
@@ -19,6 +19,18 @@ type InArgs struct {
 	Version               *bool
 }
 
+var flagHasValue = map[string]bool{
+	"vpc-config":        true,
+	"vpc-config-second": true,
+	"output-file":       true,
+	"format":            true,
+	"analysis-type":     true,
+	"grouping":          false,
+	"vpc":               true,
+	"debug":             false,
+	"version":           false,
```

Comment:
The strings of each arg are now duplicated in this file.. convert to const strings instead